opwen_email_server/api/client_write.py

```diff
@@ -32,13 +29,7 @@ def __call__(self, client_id: str, upload_info: dict) -> Tuple[str, int]:
         resource_id = upload_info.get('resource_id')
         resource_container = upload_info.get('resource_container')
 
-        QUEUE.enqueue({
-            '_version': '0.1',
-            '_type': 'lokole_emails_received',
-            '_resource_type': resource_type,
-            'resource_id': resource_id,
-            'container_name': resource_container,
-        })
+        result = store.delay(resource_id)
```

Comment:
Given that we're not doing anything with the `result`, might as well delete the variable.

Reply:
👍🏽